docs/02_cleos/02_how-to-guides/how-to-stake-resource.md

```diff
@@ -1,28 +1,110 @@
-## Goal
+## Overview
 
-Stake resource for your account
+This how-to guide provides instructions on how to stake resources, NET and/or CPU, for your account using the `cleos` CLI tool.
 
 ## Before you begin
 
-* Install the currently supported version of `cleos`
+* Install the currently supported version of `cleos`.
 
-* Ensure the reference system contracts from `eosio.contracts` repository is deployed and used to manage system resources
+* Ensure the [reference system contracts](https://developers.eos.io/manuals/eosio.contracts/v1.9/build-and-deploy) are deployed and used to manage system resources.
 
 * Understand the following:
-  * What is an account
-  * What is network bandwidth
-  * What is CPU bandwidth
+  * What is an [account](https://developers.eos.io/welcome/latest/glossary/index/#account).
+  * What is [NET bandwidth](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/net).
+  * What is [CPU bandwidth](https://developers.eos.io/manuals/eosio.contracts/latest/key-concepts/cpu).
```

Comment:
Since these are not questions, the correct sentence construction should be:
* What an [account] is.
* How [NET bandwidth] is.
* What [CPU bandwidth] is.

Another option is just to convert them to questions by appending a question mark (?) at the end of each bullet.

Reply:
updated.